src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,53 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    public static final DateTimeFormatter DATA_FORMATTER
+            = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
+    private static final int DATE_INDEX = 0;
+    private static final String SEPARATOR = " ";
+    private static final int NAME_INDEX = 1;
+    private static final int HOUR_INDEX = 2;
+    private static final int WAGE_INDEX = 3;
+    private static final String DASH = " - ";
```

Comment:
use the constant name as an opportunity to make its role in the program clear, naming dash as DASH doesn't really make it add any new information :)
in this app I'd say it's okay to use string literal in the stringBuilder itself, or you could use a template (String.format).  BTW you don't use it on the line 28, but only on 48